cpp/include/raft/core/mdarray.hpp

```diff
@@ -712,10 +823,7 @@ auto make_host_matrix(size_t n_rows, size_t n_cols)
 template <typename ElementType, typename LayoutPolicy = layout_c_contiguous>
 auto make_device_matrix(size_t n_rows, size_t n_cols, rmm::cuda_stream_view stream)
```

Comment:
As @trivialfis pointed out, I'd prefer to use the handle here as well. However, for now I'm okay matching what's there and in the future we might scrape through and either add overloads for the handle or change the stream argument to the handle.